src/gui_common/TranslationSiteButton.cs

```diff
@@ -0,0 +1,14 @@
+using Godot;
+using System;
+
+/// <summary>
+///   Shows a version label
+/// </summary>
+public class TranslationSiteButton : Button
+{
+    public override void _Process(float delta)
+    {
+        if (Pressed)
+            OS.ShellOpen("https://translate.revolutionarygamesstudio.com/");
```

Comment:
I'd prefer to use https://translate.revolutionarygamesstudio.com/engage/thrive/ because it is a little more friendly for people just starting out